src/transformState.js

```diff
@@ -4,8 +4,39 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
-function transformState(state, actions) {
-  // write code here
+function modifyState(state, actions) {
+  for (const action of actions) {
+    const { type } = action;
+
+    switch (type) {
```

Comment:
It would be better to assign action type inside the switch statement
switch (action.type)